### ISSUE TITLE
Email template examples

### ISSUE THREAD

**Issue by philippeluickx** [obsidiancard/django-allauth-djrill#6]
Hey!

Looking at https://github.com/pennersr/django-allauth/tree/master/allauth/templates/account/email
It is not really clear to me what the 4 email templates are that you are referring to.

AFAIK the password_reset_key is used when reseting the password, while email_confirmation_signup is used when a user needs to verify his email. Not sure what email_confirmation_welcome or email_confirmation are used for?

Maybe also show some examples including the merge_var parameters?

**Comment by tarkatronic** [comment]
Unfortunately, this leads back to poor documentation in django-allauth itself, as you've pointed out. You are correct that `password_reset_key` is used when resetting the password, although I had to check the allauth code to be sure of that. There are some strange distinctions between the others which I can't recall off the top of my head.

I'll have to dig through django-allauth again to figure out the precise usage/intent for each of these templates.

As far as examples including merge_vars, are you talking about calling `tasks. send_mandrill_template_mail` directly?

**Comment by philippeluickx** [comment]
Yeah, the all_auth implementation is a bit complex as there are nested emails. What I understood is that in the end there is only 1 (not 3) email generated.

For the merge_vars: I am talking about the variables that are being sent to mandrill. So e.g. *|activate_url|*.
